EdvClient.js

```diff
@@ -553,11 +553,10 @@ export class EdvClient {
    * @param {Object} options.invocationSigner - An API with an
    *   `id` property and a `sign` function for signing a capability invocation.
    *
-   * @return {Promise<Object>} resolves once the operation completes.
+   * @returns {undefined} resolves once the operation completes.
```

Comment:
This function is `async` so will return a `Promise`.  Looks like [they suggest](https://jsdoc.app/tags-async.html) using the `@async` tag.  It seems unclear what to use when there's no result value, maybe `@return {Promise} ...`?

Reply:
undefined is what the promise will resolve to so `Promise<undefined>` is what I will use. https://github.com/digitalbazaar/edv-client/pull/26/commits/2f851b4f632828632ca44c6f5bcd8cf2712abd83